api/config/es_sidecar/config_request.go

```diff
@@ -141,4 +67,173 @@ func (c *ConfigRequest) SetGlobalConfig(g *ac.GlobalConfig) {
 	if logFormat := g.GetV1().GetLog().GetFormat().GetValue(); logFormat != "" {
 		c.V1.Sys.Log.Format.Value = logFormat
 	}
+
+	//
+	// Users can configure the ElasticSearch backup configuration the at es-sidecar
+	// level, the global backups level, and in some cases the global external
+	// level. Precedence for configuration is as follows in order of priority
+	//
+	// * External configuration (if using external ElasticSearch)
+	// * Global configuration
+	// * User defined es-sidecar configuration
+	// * es-sidecar default configuration
+	//
+	externalCfg := g.GetV1().GetExternal().GetElasticsearch()
+
+	backupsCfg := c.GetV1().GetSys().GetBackups()
+	if backupsCfg == nil {
+		backupsCfg = &ConfigRequest_V1_System_Backups{}
+	}
+
+	// If we're in external mode, apply and global and external config
+	if externalCfg.GetEnable().GetValue() {
+		c.applyGlobalBackupConfig(backupsCfg, g.GetV1().GetBackups())
```

Comment:
can this be pulled out of the if statement? it looks like its done first in both branches